Source/Plugins/Core/com.equella.core/src/com/tle/web/sections/standard/renderers/ImageRenderer.java

```diff
@@ -100,9 +100,13 @@ protected void prepareFirstAttributes(SectionWriter writer, Map<String, String>
     attrs.put("src", source); // $NON-NLS-1$
     attrs.put("width", width); // $NON-NLS-1$
     attrs.put("height", height); // $NON-NLS-1$
-    if (alt != null) {
+    if (alt != null && alt.getText().length() > 0) {
```

Comment:
I'm thinking there's nothing (at least in the doco for the Label interface) that says `getText()` will return a non-null value. That combined with convention elsewhere in the code base could I suggest using `StringUtils`:

```suggestion
    if (alt != null && StringUtils.isNotBlank(alt.getText())) {
```